quickfixj-codegenerator/src/main/java/org/quickfixj/codegenerator/MessageCodeGenerator.java

```diff
@@ -305,10 +305,6 @@ private void generateCodeFile(Task task, Document document, Map<String, String>
             if (!task.isOverwrite()) {
                 return;
             }
-            if (outputFile.lastModified() > task.getSpecificationLastModified()) {
```

Comment:
@david-gibbs-ig any idea why this was removed? Now the code is generated on each compile run, regardless if it was already created in a former run.